GitUI/Editor/FindAndReplaceForm.cs

```diff
@@ -401,11 +408,58 @@ protected override void Dispose(bool disposing)
 
             base.Dispose(disposing);
         }
+
+        internal TestAccessor GetTestAccessor() => new TestAccessor(this);
+
+        internal readonly struct TestAccessor
+        {
+            private readonly FindAndReplaceForm _control;
+
+            public TestAccessor(FindAndReplaceForm control)
+            {
+                _control = control;
+            }
+
+            public TestAccessor SetCriteria(string lookFor, bool matchCase = false, bool matchWholeWordOnly = false)
```

Comment:
The idea behind `TestAccessor` was provide access to private type members without increasing their access level, and not really act as a wrapper type. This is so we can easily replace and remove this implementation with another (which is coming) without (significant) changes to tests.

Let's change this to be something like:
```cs
public TextEditorSearcher _search => _control._search;
public TextBox txtLookFor => _control.txtLookFor;
public CheckBox chkMatchCase => _control. chkMatchCase;
public CheckBox chkMatchWholeWord => _control.chkMatchWholeWord;
```

Reply:
Got it. Done.